modules/salt/pages/salt-states.adoc

```diff
@@ -164,3 +165,65 @@ Apply the highstate to all minions in the group:
 ----
 salt -I 'group_ids:GID' state.apply
 ----
+
+
+
+[[salt.pillars.setpackageendpoint]]
+== Use pillars to set the package download endpoint
+
+By default, {productname} assumes that the download endpoint to use is the FQDN of the {productname} server, or the {productname} Proxy.
+However, there are some cases where you might like to use a different FQDN as the download endpoint.
+The most common example is if you need to use load balancing, caching proxies, or in environments with complicated networking requirements.
+
+To change the package download endpoint, you can manually adjust three salt pillars:
+* [systemitem]``pkg_download_point_protocol``, defaults to [systemitem]``https``.
+* [systemitem]``pkg_download_point_host``, defaults to the FQDN of the {productname} Server (or Proxy, if in use).
+* [systemitem]``pkg_download_point_port``, defaults to [systemitem]``443``.
+
+If you do not adjust these pillars directly, {productname} will fall back to the default values.
+
+.Procedure: Changing the package download endpoint pillar
+
+. Navigate to [path]``/srv/pillar/`` and create a file called [filename]``top.sls``  with these contents:
++
+----
+base:
+  '*':
+    - rpm_download_points
+----
++
+This example directs Salt to look at the [filename]``rpm_download_points.sls`` file to determine the base URL to use.
+You can adjust this file to target different minions or groups, depending on your environment.
++
+. Remain in [path]``/srv/pillar/`` and create a file called [filename]``rpm_download_points.sls`` with the base URLs you want to use.
+For example:
++
+----
+rpm_download_point_protocol: http
+rpm_download_point_host: example.com
+rpm_download_point_port: 444
+----
+. OPTIONAL: If you want to use external pillars, for example Group IDs, open the master configuration file and set the [systemitem]``ext_pillar_first`` parameter to [systemitem]``true``.
+You can then Group IDs to set conditional values, for example:
```

Comment:
Missing 'use' word I guess. Moreover, It would be nice if you could move the example with grains before the Pilllars one.